spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-simple-core/src/test/java/org/springframework/cloud/kubernetes/fabric8/core/SimpleCoreIT.java

```diff
@@ -91,64 +85,25 @@ void test() {
 		Assertions.assertEquals("Hello from k8s profile", result);
 	}
 
-	private static void deleteManifests() {
+	private static void manifests(Phase phase) {
 
-		try {
```

Comment:
@ryanjbaxter here is the idea of my PR, for example. In almost all of our integration tests, we create a deployment,service,ingress (and may be configmaps and secrets), and they do the exact same thing. First create them, do the testing, then deleted them. So why can't we simplify this and hide all the complexity of waiting for the resources to be created and deleted via proper methods? In such a case, reading the code of the integration test becomes far simpler to reason about. I've done this in all the tests + did some clean-up here and there.